tests/Unit.php

```diff
@@ -2,16 +2,86 @@
 
 class Unit extends PHPUnit_Framework_TestCase {
 
-	public function subject($params = array(), $options = array()) {
-		$options += array(
-			'mock' => false,
-			'methods' => array(),
+	public static $classNamespace = "alfmarks\\";
+
+	private $subjectClass;
+	private $subjectParams = array();
+	private $subjectOptions;
+
+	private $baseParams = array();
+
+	private $baseOptions = array(
+		'mock' => false,
+		'methods' => array()
+	);
+
+	public function __construct( ...$class ) {
```

Comment:
This ellipses isn't used, and you'd overwrite `subjectClass` each time you have multiple arguments. You could do something like:

~~~ php
public function __construct() {
  foreach (func_get_args() as $class) {
    $this->subjectClass = $klass;
  }
}
~~~